src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/InvalidHwFCallbackHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.citizenui.CaseDataLiP;
+import uk.gov.hmcts.reform.civil.model.citizenui.HelpWithFees;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+
+@ExtendWith(MockitoExtension.class)
+public class InvalidHwFCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @InjectMocks
+    private InvalidHwFCallbackHandler handler;
+
+    @Mock
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Test
+    void shouldSuccessfullyCreateEvent() {
+        CaseData caseData = CaseDataBuilder.builder()
+            .caseDataLip(CaseDataLiP.builder()
+                             .helpWithFees(HelpWithFees.builder()
+                                               .helpWithFeesReferenceNumber("123")
+                                               .build())
+                             .build())
+            .build();
+
+        CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        assertThat(response.getErrors()).isNull();
```

Comment:
Add assert to test main logic which is I think to update InvalidHelpWithFee value